libs/ccc/utils/utility_functions.py

```diff
@@ -7,9 +7,12 @@
 from subprocess import run
 
 import numpy as np
-import pandas as pd
 
-from .log import get_logger
+# optional packages to avoid problems with the CCC PyPI package
+try:
+    from ccc.log import get_logger
+except ModuleNotFoundError:
+    pass
```

Comment:
Should there also be a check downstream when you try to use `get_logger`?

Reply:
I've added a comment here to explain more. Hopefully, now it is explained better. It will never happen that the `get_logger` function is not found/loaded by the code that needs it. Let me know if you still have questions/comments.

```python
# optional packages to avoid problems with the CCC PyPI package
try:
    # the "ccc.log" module is only used by the code that performs the analyses
    # for the manuscript, but it is not used in the ccc-coef package (PyPI)
    # although ccc-coef uses some functions in this module (ccc.utils). Since
    # "ccc.log" is not distributed as part of ccc-coef, I need to check if it is
    # present and ignore any ModuleNotFoundError
    from ccc.log import get_logger
except ModuleNotFoundError:
    pass
```